projects/storefrontlib/cms-components/product/carousel/product-carousel/product-carousel.component.spec.ts

```diff
@@ -113,6 +114,7 @@ describe('ProductCarouselComponent', () => {
         imports: [RouterTestingModule, I18nTestingModule],
         declarations: [
           ProductCarouselComponent,
+          ProductCarouselItemComponent,
```

Comment:
Mock this component and move element-specific tests to product-carousel-item.component.spec.ts